### PR TITLE
Updated missing periods in cmdline.rst

### PR DESCRIPTION
**Pull request by Seluj78**

### CODE REVIEW

File: Doc/using/cmdline.rst [hunk 1]
```diff
@@ -898,7 +898,7 @@ conflict.
 .. envvar:: PYTHONUTF8
 
    If set to ``1``, enables the interpreter's UTF-8 mode, where ``UTF-8`` is
-   used as the text encoding for system interfaces, regardless of the
+   used as encoding for system interfaces, regardless of the
```
Comment (Seluj78) [inline Doc/using/cmdline.rst]:
```suggestion
   used as the text encoding for system interfaces, regardless of the
```
Oupsy

**Comment by Seluj78** [comment]
Ping @vstinner and @JulienPalard

**Review by brandtbucher — APPROVED** [review]
Thanks for the PR @Seluj78!

**Comment by vstinner** [comment]
@Seluj78: the automated backport to 3.7 failed. I expect a conflict. Would you like to backport manually the change.

**Comment by Seluj78** [comment]
@vstinner No problemo, I will do it manually ! 😁

**Pull request merged**